### PR TITLE
fix: local superset-sso working

### LINKED ISSUE #71: Incorrect default Superset SSO redirect URL
In tutor local (at least) on a fresh install, trying to log into superset gives:

```
Error: invalid_request

Mismatching redirect URI.
```

The redirect url for the superset-sso application is incorrectly set to: http://superset.local.overhang.io:8088/oauth-authorized/openedxsso . Removing the port fixes the problem.

### PR DESCRIPTION
**Pull request by Ian2012**
Fixes #71 

image

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoraspects/patches/local-docker-compose-dev-services [hunk 1]
```diff
@@ -2,32 +2,23 @@
...
     OPENEDX_LMS_ROOT_URL: "http://{{ LMS_HOST }}:8000"
+    OAUTH2_CLIENT_ID: {{ SUPERSET_OAUTH2_CLIENT_ID_DEV }}
   command: ["bash", "/app/docker/docker-bootstrap.sh", "app-gunicorn"]
   ports:
     - 8088:{{ SUPERSET_PORT }}
   depends_on:
     - mysql
     - redis
     - superset-worker
-    - superset-worker-beat
```
Comment (bmtcril) [inline tutoraspects/patches/local-docker-compose-dev-services:12]: How come the beat container is being removed? Seems like a thing we might want for alerts and emailing reports on a schedule?

Comment (Ian2012) [reply tutoraspects/patches/local-docker-compose-dev-services:12]: This beat service was just another worker. The beat command should be 
`"bash", "/app/docker/docker-bootstrap.sh", "beat"`

Do we need this for local development if we already have worker and the beat is just eating resources?

Comment (Ian2012) [reply tutoraspects/patches/local-docker-compose-dev-services:12]: Forget it, I see that this is needed for alerts

Comment (Ian2012) [reply tutoraspects/patches/local-docker-compose-dev-services:12]: @bmtcril I did found out that this service was not running in k8s, so i also added support for it in k8s

**Review by bmtcril — APPROVED** [review]
LGTM, thanks!

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-oars&entry.867055334=https://github.com/openedx/tutor-contrib-oars/pull/87&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-06-23+17:43&entry.1917517419=2023-06-23+18:56&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**